easybuild/easyconfigs/r/RELION/RELION-3.0.7-intel-2019a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '3.0.7'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/3dem/relion/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['385ae3ae992cef60aa4ce97075dd4580bf00172154af426adf5ec509e248daac']
+
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [
+    ('X11', '20171023'),
+    ('FLTK', '1.3.4'),
+    ('LibTIFF', '4.0.9'),
+    ('tbb', '2019_U4'),
+]
+
+configopts = "-DCMAKE_SHARED_LINKER_FLAGS='-lpthread'  -DMPI_INCLUDE_PATH=$EBROOTIMPI/intel64/include "
+configopts += "-DMPI_C_COMPILER=$EBROOTIMPI/intel64/bin/mpiicc -DMPI_CXX_COMPILER=$EBROOTIMPI/intel64/bin/mpiicpc "
```

Comment:
Didn't think of that before, but the way to do this is to specify the 'usempi': True toolchain option.
Then you don't need to set those MPI_xxx variables.